packages/react-aria-components/src/Toast.tsx

```diff
@@ -94,12 +104,28 @@ export const ToastRegion = /*#__PURE__*/ (forwardRef as forwardRefType)(function
     : null;
 });
 
-// TODO: possibly export this so additional children can be added to the region, outside the list.
-const ToastList = /*#__PURE__*/ (forwardRef as forwardRefType)(function ToastList<T>(props: ToastRegionProps<T>, ref: ForwardedRef<HTMLOListElement>) {
+export interface ToastListProps<T> extends Omit<ToastRegionProps<T>, 'queue' | 'children'> {
+  /** A function to render each toast. */
+  children: (renderProps: {toast: QueuedToast<T>}) => ReactElement
+}
+
+export const ToastList = /*#__PURE__*/ (forwardRef as forwardRefType)(function ToastList<T>(props: ToastListProps<T>, ref: ForwardedRef<HTMLOListElement>) {
```

Comment:
Can we have a test for the newly split out ToastList setup? It could be a chromatic of the s2 Toast, so long as there is something so we know if S2 Toasts stopped working catastrophically. We should get some actual s2 tests as well.

Reply:
yeah I guess we wanted to get this into a release as unstable for now, and I'll definitely be following up with a bunch more stuff including tests.